src/highlight/highlight.ts

```diff
@@ -1,4 +1,4 @@
-import get from "lodash-es/get";
+import { get } from "lodash";
```

Comment:
does this import correctly or add the whole lodash to the library?

Reply:
I changed it, now we use same logic as instaantsearch.js so better than before I guess